Remotion/Data/DomainObjects.UnitTests/Tracing/TracingDbCommandTest.cs

```diff
@@ -45,14 +52,39 @@ public void SetUp ()
     [Test]
     public void Dispose ()
     {
-      _innerCommandMock.Setup(mock => mock.Dispose()).Verifiable();
+      _innerCommandMock.Protected().Setup("Dispose", [true]).Verifiable();
 
       _command.Dispose();
 
       _innerCommandMock.Verify();
       _extensionMock.Verify();
     }
 
+    [Test]
+    public void Dispose_NotDisposing ()
+    {
+      _innerCommandMock.Protected().Setup("Dispose", [true]).Verifiable();
+
+      var method = _command.GetType().GetMethod("Dispose", BindingFlags.NonPublic | BindingFlags.Instance);
+      method.Invoke(_command, [false]);
+
+      _innerCommandMock.Protected().Verify("Dispose", Times.Never(), [true]);
```

Comment:
```suggestion
      _innerCommandMock.Protected().Verify("Dispose", Times.Never(), [Arg<bool>.Is.Anything]);
```
Please check syntax.